File: CStoreManagement/app/src/main/java/com/cstore/zhiyazhang/cstoremanagement/view/TestActivity.java
package com.cstore.zhiyazhang.cstoremanagement.view;

import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.widget.Button;

import com.cstore.zhiyazhang.cstoremanagement.view.instock.scrap.ScrapActivity;

/**
 * Created by zhiya.zhang
 * on 2017/10/17 14:54.
 */

public class TestActivity extends AppCompatActivity {
    Button btn;
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        btn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(TestActivity.this, ScrapActivity.class));
            }
        });

    }
}

File: CStoreManagement/app/src/main/java/com/cstore/zhiyazhang/cstoremanagement/view/TestActivity.kt
package com.cstore.zhiyazhang.cstoremanagement.view

import com.cstore.zhiyazhang.cstoremanagement.R
import com.cstore.zhiyazhang.cstoremanagement.utils.MyActivity
import kotlinx.android.synthetic.main.activity_test.*

/**
 * Created by zhiya.zhang
 * on 2018/3/20 11:25.
 */
class TestActivity(override val layoutId: Int = R.layout.activity_test) : MyActivity() {
    override fun initView() {
        scroll.post {
            scroll.smoothScrollTo(0,linear.measuredHeight)
        }
    }

    override fun initClick() {
    }

    override fun initData() {
    }

}